src/scenes/Activities/ActivityHistory/index.js

```diff
@@ -104,9 +107,14 @@ export default class ActivityHistory extends Component {
                 />
               </Table>
             ) : (
-              <Typography variant="body1">
-                <FormattedMessage id="str.emptyTxHistory" defaultMessage="You do not have any transactions right now." />
-              </Typography>
+              <Row>
```

Comment:
Move the whole placeholder out of Grid. It will leads inconsistent of position with other pages.